Content.Server/Voting/VoteCommands.cs

```diff
@@ -92,6 +108,40 @@ public void Execute(IConsoleShell shell, string argStr, string[] args)
                 options.Options.Add((args[i], i));
             }
 
+            // Set up the webhook payload
+            var _gameTicker = _entitySystem.GetEntitySystem<GameTicker>();
+
+            var payload = new WebhookPayload()
```

Comment:
I guess use that before creating and sending the webhook
```cs
if (!String.IsNullOrEmpty(webhook))
    {
    // code
    }

Reply:
Using it there caused issues, had it in the method instead